### PR TITLE
Default label for unnamed printer

### LINKED ISSUE #82: Show some kind of default label if a printer is unnamed
Right now we just show a blank if a printer is unnamed, but we should show some kind of label in the sidebar and in the printer page. Maybe just "Unnamed Printer"

<bountysource-plugin>

---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/57058966-show-some-kind-of-default-label-if-a-printer-is-unnamed?utm_campaign=plugin&utm_content=tracker%2F60188248&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F60188248&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by kmigielek**
Fixes #82

### CODE REVIEW

File: src/Objects/Printer.vala [hunk 1]
```diff
@@ -167,7 +167,8 @@ public class Printers.Printer : GLib.Object {
      */
     public string info {
         get {
-            return CUPS.get_option ("printer-info", dest.options) ?? "";
+            unowned string? info = CUPS.get_option ("printer-info", dest.options);
+            return info != null && info != "" ? info : "Unnamed printer";
```
Comment (donadigo) [inline src/Objects/Printer.vala]: You need to make the string translatable and have "printer" title case.
```suggestion
            return info != null && info != "" ? info : _("Unnamed Printer");
```

Comment (kmigielek) [reply src/Objects/Printer.vala]: Changes applied

**Comment by peteruithoven** [comment]
This feels a bit double? Looking at how the "null coalescing operator" (`??`) works in: https://wiki.gnome.org/Projects/Vala/Tutorial/ I understand that if `CUPS.get_option ("printer-info", dest.options)` returns `null` then `""` will be used. Which I guess means that the `info != null` check is not needed. 

When `get_option` always returns `null` when a printer is unnamed (I'm not sue that's true, please verify), then the following should even be enough?
``` 
return CUPS.get_option ("printer-info", dest.options) ?? _("Unnamed Printer");
```

**Comment by kmigielek** [comment]
> This feels a bit double? Looking at how the "null coalescing operator" (`??`) works in: https://wiki.gnome.org/Projects/Vala/Tutorial/ I understand that if `CUPS.get_option ("printer-info", dest.options)` returns `null` then `""` will be used. Which I guess means that the `info != null` check is not needed.
> 
> When `get_option` always returns `null` when a printer is unnamed (I'm not sue that's true, please verify), then the following should even be enough?
> 
> ```
> return CUPS.get_option ("printer-info", dest.options) ?? _("Unnamed Printer");
> ```

**Comment by peteruithoven** [comment]
@kmigielek thank you! 

Looks good to me, but there is a running review which means I can't approve.

**Pull request merged**